src/include/timestamp_inline.h

```diff
@@ -76,6 +76,7 @@
         (dest)->durable_start_ts = (source)->durable_start_ts; \
         (dest)->start_ts = (source)->start_ts;                 \
         (dest)->start_txn = (source)->start_txn;               \
+        (dest)->prepare = (source)->prepare;                   \
```

Comment:
Nice catch - could this have broader consequences.
@kommiharibabu and @bvpvamsikrishna - it's worth noting this change.

Reply:
I find that though we have this macro already but no code is using it before so there is no consequence of this change.

Reply:
No. The macro `WT_TIME_WINDOW_COPY_START` is used only in fixing the history store update with the stop timestamp and history store updates do not contains prepared updates.

Reply:
@kommiharibabu Thanks I missed that usage.